docs/source/contributors/index.md

```diff
@@ -117,3 +117,41 @@ To uninstall your Jupyter AI development environment, deactivate and remove the
 conda deactivate
 conda env remove -n jupyter-ai
 ```
+
+## Testing
+
+### Integration / E2E tests
+
+This extension uses Playwright for the integration / E2E tests (user-level tests).
+More precisely, the JupyterLab helper
+[Galata](https://github.com/jupyterlab/jupyterlab/tree/master/galata) is used to
+test the extension in JupyterLab.
+
+Install test dependencies (needed only once):
+
+```sh
+cd ./packages/jupyter-ai/ui-tests/
+jlpm install
+jlpm playwright install
+cd ../../../
```

Comment:
I think it is ok to start with cd into the ui-tests directory in each section, but is repetitive with each cd ../../../ and the cd again in the ui-tests directory.